### PR TITLE
feat(frontity): expose fetch and URL from frontity package

### LINKED ISSUE #166: Expose "fetch" and "URL" in node global
**Describe the expected behavior**

`fetch` and `URL` should be available on node `global`.

**Describe the observed behavior**

Right now they are not.

We are using `cross-fetch` but it adds 15kbs of code.

**Solution**

We can import `URL` from the node module `"url"` and fetch from `"node-fetch"`.
Then import them in `/server/index.ts` like this:
```js
(global as any).fetch = fetch;
(global as any).URL = URL;
```

_It will work on TypeScript because it thinks they come from window._

Also, we should get rid of `cross-fetch` on `wp-source` and the regexps that are not using `URL` yet.

### PR DESCRIPTION
**Pull request by DAreRodz**
#### Description of what you did:

Closes #166

#### My PR is a:

- [ ] 🐞 Bug fix
- [ ] 🚀 New feature
- [x] 🔝 Improvement

#### Main update on the:

- [ ] Documentation
- [x] Framework

#### Is the PR ready to be merged?

- [x] Yes!
- [ ] Work in progress

### CODE REVIEW

File: packages/wp-source/package.json [hunk 1]
```diff
@@ -27,13 +27,13 @@
     "@types/jest": "^24.0.11",
     "frontity": "^1.2.2",
     "jest": "^24.7.1",
+    "node-fetch": "^2.6.0",
```
Comment (luisherranz) [inline packages/wp-source/package.json:30]: This shouldn't be in devDependencies or it won't be installed when `wp-source` is installed as a dependency.

File: packages/wp-source/src/libraries/api.ts [hunk 2]
```diff
@@ -1,5 +1,4 @@
-import fetch from "cross-fetch";
-
+import { fetch } from "frontity";
 class Api {
```
Comment (luisherranz) [inline packages/wp-source/src/libraries/api.ts:3]: Missing empty line between `import` and `class` :)

**Review by luisherranz — CHANGES_REQUESTED** [review]
Looks good but node-fetch should be added as a dependency.

File: packages/wp-source/package.json (hunk 1 above)
Comment (DAreRodz) [reply packages/wp-source/package.json:30]: Well, `node-fetch` is only needed for tests, and for the other cases `wp-source` uses `fetch` from `frontity` package. That's the reason I left `node-fetch` in `devDependencies`.

File: packages/wp-source/src/libraries/api.ts (hunk 2 above)
Comment (DAreRodz) [reply packages/wp-source/src/libraries/api.ts:3]: Oops!

**Comment by luisherranz** [comment]
By the way, "Implement" doesn't look like a GitHub word. You have to use "Fixes" or "Closes": https://help.github.com/en/articles/closing-issues-using-keywords

File: packages/wp-source/package.json (hunk 1 above)
Comment (luisherranz) [reply packages/wp-source/package.json:30]: Ohh, absolutely. But then, it should be added in the `frontity` dependencies, right?

Comment (DAreRodz) [reply packages/wp-source/package.json:30]: @luisherranz yeah, you are right. I has been there for a while, though 😅

https://github.com/frontity/frontity/blob/bd8c6dc3d9872f1780e395c952737c867c88c0e8/packages/frontity/package.json#L48

If there's nothing else to change we can resolve this and merge the PR.

Comment (luisherranz) [reply packages/wp-source/package.json:30]: Nothing else on my side! Thanks @DAreRodz.

**Review by luisherranz — APPROVED** [review]

**Pull request merged**